config/config.aws.reference.hocon

```diff
@@ -53,15 +53,13 @@
       # -- For a S3 bucket, the uri should start with `s3a://`
       "location": "s3a://my-bucket/events
 
-      # -- Atomic columns which should be brought to the "left-hand-side" of the events table, to
-      # -- enable Delta's Data Skipping feature.
-      # -- The Delta table property `delta.dataSkippingNumIndexedCols` will be set to the size of the list
-      "dataSkippingColumns": [
-        "load_tstamp"
-        "collector_tstamp"
-        "derived_tstamp"
-        "dvce_created_tstamp"
-      ]
+      # -- Any valid Delta table property
+      # -- This can be blank in most setups because the loader already sets sensible defaults.
+      "deltaTableProperties": {
+        "delta.dataSkippingStatsColumns": "load_tstamp,collector_tstamp,derived_tstamp,dvce_created_tstamp"
+        "delta.checkpointInterval": "50"
```

Comment:
I assume some of these will be exposed through a config management. The loader won't be able to sanity-check the value (and possibly some others). Are we expected to have some sort of check on how this value is set versus eg. commit frequency?